.ci-mgmt.yaml

```diff
@@ -14,6 +14,7 @@ env:
   GOOGLE_REGION: "us-central1"
   GOOGLE_ZONE: "us-central1-a"
   PULUMI_MISSING_DOCS_ERROR: true
+  VERSION_PREFIX: 7.0.0
```

Comment:
do we need this here? I think line 3 takes care of it?

Reply:
I think we do. Without this entry VERSION_PREFIX env var will not populate. Confusing pulumictl and versioning.